tests/js/client/shell/shell-keygen.js

```diff
@@ -1138,31 +1138,17 @@ function KeyGenerationLocationSuite() {
       db._drop(cn);
     },
 
-    testThatFailurePointsWork1: function () {
```

Comment:
This test has never worked, since if the failure point did not work, it was green. Now, it cannot be kept, since there should be no situation any more which triggers the single server key generator on a dbserver in a cluster.